x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/index.tsx

```diff
@@ -85,6 +85,14 @@ export function Popover({ focusedServiceName }: PopoverProps) {
     }
   }, [popoverRef, x, y]);
 
+  const centerSelectedNode = useCallback(() => {
+    if (cy) {
+      cy.center(cy.getElementById(selectedNodeServiceName));
```

Comment:
Instead of `center`, we should animate it with `cy.animate({ center: { eles: cy.getElementById(selectedNodeServiceName) } })`

Reply:
Right now animations are being initiated between data load, so it gets a little jumpy. we'll have to fix it in another PR, but this makes sense to animate right now.